lib/clientv2/networkpolicy_e2e_test.go

```diff
@@ -77,20 +77,28 @@ var _ = testutils.E2eDatastoreDescribe("NetworkPolicy tests", testutils.Datastor
 			be.Clean()
 
 			By("Updating the NetworkPolicy before it is created")
+			var rv string
+			if config.Spec.DatastoreType != apiconfig.Kubernetes {
```

Comment:
A bit sad that we need the special casing here, though I suppose it's just for this specific test.

etcd complains if you pass a RV with a `/`?

Reply:
meh - if it was a different datastore we might need to pass in another format of the RV so it doesn't feel too icky to me.